### DISCUSSION TITLE
Welcome to build-discarder-plugin Discussions!

### DISCUSSION THREAD

**Discussion by orjan** [jenkinsci/build-discarder-plugin#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ROSMCCANN** [comment]
In this plugin which one takes precedence between days and number of builds?

**Reply by orjan** [reply]
Not sure to be honest, it's using the same logic as the glvoal build discarder in Jenkins.

The only difference is that this plugin respects if there's a custom discarder specified at the job level and won't discard any builds if you have a specific one.

It's perfectly fine to combine this discarder with the one built.
You can use the default one to delete everything that's older than `100 days`.
You can set this one to delete everything that's older than `7 days`.

Then you can override on job level to `30 days`

If you're overriding on job level to `365 days` the built in discarder will kick in after `100 days`

**Comment by andreisuslov** [comment]
In logRotator, can we use `artifactDaysToKeep` instead of `artifactDaysToKeepStr`? Why String?
